README.md

```diff
@@ -1,99 +1,64 @@
-# backstage-operator
-Operator for deploying Backstage for Janus-IDP.
+# Backstage Operator
 
-## Description
-Implementing https://janus-idp.io/docs/deployment/k8s/ procedure
-At first stage CR update does not affect Backstage Objects, just installation (same as Helm)
-TODO: Do we need to continuosly sync the states? Which way if so: from CR to Objects or back or (somehow) back and forth?
+## The Goal
+The Goal of [Backstage](https://backstage.io) Operator project is creating Kubernetes Operator for configuring, installing and synchronizing Backstage instance on Kubernetes/OpenShift. 
```

Comment:
```suggestion
The Goal of this project is to create an [OperatorSDK](https://sdk.operatorframework.io/) / [OLM](https://docs.openshift.com/container-platform/4.14/operators/understanding/olm/olm-understanding-olm.html) compliant operator for deploying Backstage instances to a Kubernetes cluster.
```

Reply:
As discussed, OLM is good to have upstream but not the goal.